Source/Controller/HelpDrawerCoordinator.swift

```diff
@@ -178,19 +203,20 @@ enum HelpDrawerCoordinator {
     }
     
     // swiftlint:disable function_body_length
-    /// Builds the SwiftUI help drawer view for the given view controller.
-    @MainActor private static func helpDrawerView(
-        for viewController: HelpDrawerHost,
```

Comment:
I think it might be good to make the new HomeView and Hashtag view conform to `HelpDrawerHost` to keep this function from being tightly coupled to the `mainViewController`. Probably the best way to do it would be to rename the current `HelpDrawerHost` protocol to be `HelpDrawerViewControllerHost` or something. Then `HelpDrawerHost` could have the properties common to all hosts: `shouldUseDrawer` and `helpDrawerType`. What do you think?

Reply:
👌